### PR TITLE
chore: remove nuxt proxy module and replace with server routes

### PR DESCRIPTION
**Pull request by connoratrug**
Remove the nuxt-proxy plugin and replace dev proxy by using nuxt server routes

nuxt-proxy is dependent on a old version of @nuxt/kit, the nuxt/kit module is dependent on node v18. 
We want to keep using stable node releases ( especially then using as runtime for nuxt ).
 By removing the nuxt-proxy module the emx apps build is no longer locked to node version v18

To replace the nuxt-proxy development proxy this PR adds server routes for the proxied endpoints ( /gql , /file and /message). Still need baseURL to tell server-side nuxt where to fetch the data 

test:

- [x] app dev ( gql, file, message)
- [x] app build ( gql, file, message)
- [x] preview ( helm config)
- [x] VM run 
- [ ] contact form

### CODE REVIEW

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<template>
+  <h1>test page</h1>
+  <!-- {{ data }} -->
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue:3]: what is purpose of this test page?

File: apps/nuxt3-ssr/pages/[schema]/test.vue [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <h1>test schema page</h1>
+  {{ data }}
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/test.vue:3]: and this one?

File: apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { joinURL } from "ufo"; 
+
```
Comment (mswertz) [inline apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts:2]: this file has a very strange name

**Review by mswertz — APPROVED** [review]
just some 'test' pages that I was not sure you want to keep. 
I don't mind

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue (hunk 1 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue:3]: o, forgot to push delete

File: apps/nuxt3-ssr/pages/[schema]/test.vue (hunk 2 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/test.vue:3]: same

File: apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts (hunk 3 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts:2]: this a wildcard (https://nuxt.com/docs/guide/directory-structure/server#catch-all-route)

File: apps/nuxt3-ssr/server/routes/[schema]/api/message.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { joinURL } from "ufo";
+export default defineEventHandler((event) => {
+  const config = useRuntimeConfig(event);
+  console.log("proxy message request : ", event.path);
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/server/routes/[schema]/api/message.ts:4]: hmm yea i thought about deleting as we dont use a 'real' logging servers ( where we can inject the log level), suggest to keep for now and remove if to noisy

File: apps/nuxt3-ssr/server/routes/[schema]/graphql.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { joinURL } from "ufo";
+export default defineEventHandler((event) => {
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/server/routes/[schema]/graphql.ts:2]: yes , but keep in mind this (currently) only runs on the (node)server, not the on client, the client still calls source ( then hit the nginx that then call the emx2 gql api, the call from the client only hits the nuxt-server routes during dev )

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue (hunk 1 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/test.vue]: done

File: apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts (hunk 3 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts:2]: done

**Pull request merged**